cranserver/server.py

```diff
@@ -91,30 +83,19 @@ def home_post():
     if 'file' not in request.files:
         return redirect(request.url)
     file = request.files['file']
-
     if file:
-        a_lock = fasteners.InterProcessLock(LOCK_FILE_LOC)
-        gotten = a_lock.acquire(
-            blocking=True, delay=0.2, timeout=LOCK_TIMEOUT)
+        pkg = Package.from_tarball(file)
+        file.seek(0)
```

Comment:
why leave this seek if we're done with `file`?

Reply:
Because you have to reset the file cursor to upload to S3.